src/pages/Models/Models.js

```diff
@@ -143,6 +133,15 @@ export default function Models() {
 
       <div className="l-content">
         <div className="models">
+          <StatusStrip
+            statusList={{
+              model: [
```

Comment:
Is there ever a time when we would have multiple keys? If not using this data structure really complicates the parsing in the component vs `<StatusStrip label="model" statuses={statuses} />`